src/eval-frame/actions/actions.js

```diff
@@ -22,10 +22,6 @@ initialVariables.add("CodeMirror");
 initialVariables.add("FETCH_RESOLVERS");
```

Comment:
We should probably break the `initialVariables` obj + associated function into a separate file as well, either in this PR or elsewhere. Mind adding a `FIXME` if you don't want to tackle this now?